app/controllers/comments_controller.rb

```diff
@@ -211,14 +211,22 @@ def check_permission_to_edit
   def check_permission_to_modify_frozen_status
     return if permission_to_modify_frozen_status
 
-    flash[:error] = t(".permission_denied")
```

Comment:
You could write `t("comments.#{action_name}.permission_denied")` and keep the `i18n-tasks-use` hints. Subjective but I'd prefer that over an if statement without else in this case.